Library/Formula/smpeg2.rb

```diff
@@ -0,0 +1,46 @@
+class Smpeg2 < Formula
+  desc "SDL MPEG Player Library"
+  homepage "http://icculus.org/smpeg/"
+  url "svn://svn.icculus.org/smpeg/tags/release_2_0_0/"
+
+  head "svn://svn.icculus.org/smpeg/trunk"
+
+  option "with-plaympeg", "Build plaympeg application"
+  if build.with? "plaympeg"
+    conflicts_with "homebrew/head-only/smpeg", :because => "both install `plaympeg` binaries."
```

Comment:
Let's move https://github.com/Homebrew/homebrew-head-only/pull/79 into this PR as well (As a separate commit) and fix any `brew audit --strict` things on that, and then we can keep the `conflicts_with`. With the release available `smpeg` doesn't really belong in `head-only` any more.